modules/special-rules/pages/psychic-beacon.adoc

```diff
@@ -1,17 +1,18 @@
 = Psychic Beacon
 
 `Detachment ability`
+footnote:[{e40k-diff}
+The {e40k} version of this ability is exceptionally cheap and powerful -- for only a small cost, it allows all Mycetic Swarm detachments in the army to arrive in turn 1, automatically.
+We've revised the ability so that it scales with the number of detachments that have the ability, and so that it improves rather than bypasses the odds on the Reserve Detachment Arrival table.
+]
```

Comment:
[I posted this comment before, but that thread became outdated and hidden during the process of merging content from the 'footnote' branch to master and then to this branch, while I also replaced the E40k difference notes with footnotes in this branch. Hence the re-post.]

This new version of the ability seems much better but: Aside from the 'non-scaling' and 'automatic' problems, the original was also exceptional in that it allowed Mycetic Swarms to arrive on the very first turn — whilst players can't start to roll for Reserves until the second turn.

Did you intend that this ability should no longer make it possible at all for Mycetic Swarms to arrive on the very first turn? (If yes, then great, that's obviously the simplest and most consistent — but it is another big difference so I thought I'd check.)

Reply:
Yes, that's the intention. Instead of using a completely unique mechanic, it's just a re-roll using the standard Reserves mechanics.

Reply:
```suggestion
footnote:[{e40k-diff}
The {e40k} version of this ability is exceptionally cheap and powerful -- for only a small cost, it allows all Mycetic Swarm detachments in the army to arrive in turn 1, automatically.
We've revised the ability so that it scales with the number of detachments that have the ability, and so that it improves rather than bypasses the odds on the Reserve Detachment Arrival table.
This also means that the earliest a Mycetic Swarm detachment can arrive from Reserves is turn 2, just like other Reserves.
]
```